packages/components/src/presentation-components/tree/controlled/PresentationTreeRenderer.tsx

```diff
@@ -155,9 +155,7 @@ function MatchingInstancesCount({ filter, dataProvider, parentKey }: MatchingIns
       } catch (e) {
         if (e instanceof PresentationError && e.errorNumber === PresentationStatus.ResultSetTooLarge) {
           // ResultSetTooLarge error can't occur if sizeLimit is undefined.
-          return `${translate("tree.filter-dialog.result-limit-exceeded")} ${requestOptions.sizeLimit!}. ${translate("tree.please-provide")} ${translate(
-            "tree.additional-filtering",
-          )}.`;
+          return translate("tree.filter-dialog.result-limit-exceeded");
```

Comment:
Should clean up other, now unused, strings from the localization file. Also, I personally would want to know what the limit is, so maybe change the message to something like this?

> There are too many results to display. Please adjust your filters to limit the results count to X items.